counsel.el

```diff
@@ -5234,6 +5236,20 @@ You may, for example, want to add \"-jN\" for the number of cores
 N in your system."
   :type 'string)
 
+(defcustom counsel-compile-env nil
+  "List of environment variables for compilation to inherit.
+Each element should be a string of the form ENVVARNAME=VALUE.  This
+list is passed to `compilation-environment'."
+  :type '(repeat (string :tag "ENVVARNAME=VALUE"))
+  :options '(("LANG=C")))
+
+(defvar counsel-compile-env-history nil
+  "History for `counsel-compile-env'.")
+
+(defvar counsel-compile-env-pattern
+  "\\(?:^[_[:digit:][:upper:]]+=[/[:alnum:]]?\\)"
```

Comment:
Why does this need to be wrapped in a shy group? Why is only a single character allowed following the equals sign?

Reply:
Sorry that should be * (ie zero or more) so an env can be unset.